gulpfile.js

```diff
@@ -354,12 +176,18 @@ const SKIP_PACKAGES = new Set([
 ]);
 const SKIP_PACKAGES_LIST = [...SKIP_PACKAGES];
 
-const pkgs = fs
-  .readdirSync('packages')
-  .filter((file) => !file.startsWith('.'))
-  .filter((file) => !SKIP_PACKAGES.has(file))
-  .filter((pkg) => fs.pathExistsSync(path.resolve('packages', pkg, 'package.json')));
-const pkgJSONs = pkgs.map((pkg) => [pkg, getPackageJSON(pkg)]);
+const getPackageJSONs = () => {
```

Comment:
Why wrap this in a function?

Reply:
So I can call it in my browserify task.  If I don't, then pkgJSONs is empty if I run the task alone.